wls-gui-wahllokalsystem/src/components/wahlhandlung/beanstandeteWahlbriefe/TheBeanstandeteWahlbriefeTable.vue

```diff
@@ -43,21 +41,22 @@
                     onZulassungsgrundWahlscheinChanged(value, index - 1)
                 "
               />
-            </v-row>
+            </div>
           </td>
           <td
             v-for="wahl in wahlenState.wahlen"
             :key="`${wahl.wahlID}-${index - 1}`"
+            class="fill-height"
```

Comment:
```suggestion
            class="fill-height wahlscheinReasonStyling"
```

<img width="931" height="241" alt="Image" src="https://github.com/user-attachments/assets/debfa78c-bb6e-4dc0-bf03-cba7f4563e38" />

ich fände es schöner wenn die width nicht nur für die wahlscheine sondern für alle spalten gelten würde:

<img width="1059" height="205" alt="Image" src="https://github.com/user-attachments/assets/f864b964-4209-4c18-afb1-d1c1cd1591c2" />

Reply:
angepasst mit f1c0721c3c2e9816615088fcccd6bb97136e46a3